Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1990,7 +1990,6 @@ extension BrowserViewController: LegacyTabDelegate {
 
     func tab(_ tab: Tab, willDeleteWebView webView: WKWebView) {
         DispatchQueue.main.async { [unowned self] in
-            tab.cancelQueuedAlerts()
```

Comment:
Why can we now remove this?

Reply:
Because the cancel func was only calling the saved completionHandler but now we are calling it from the delegate directly and not saving it anymore